presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -251,4 +252,75 @@ public static long crc32(@SqlType(StandardTypes.VARBINARY) Slice slice)
         crc32.update(slice.toByteBuffer());
         return crc32.getValue();
     }
+
+    @Description("suffix starting at given index")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice substr(@SqlType(StandardTypes.VARBINARY) Slice slice, @SqlType(StandardTypes.BIGINT) long start)
```

Comment:
I don't see much difference between these two functions actually. `length` is used to specify an ending point. We could even make `length = Long.MAX_VALUE`. Please delegate the function to the other one.

Reply:
using the accurate value length = slice.length() - start + 1.